app/helpers/invitation_codes_helper.rb

```diff
@@ -6,13 +6,13 @@ def invite_hidden_tag(invite)
   end
 
   def invite_link(invite_code)
-    text_field_tag :invite_code, invite_code_url(invite_code), :readonly => true
+    text_field_tag :invite_code, invite_code_url(invite_code), :class => 'form-control', :readonly => true
```

Comment:
Actually, I'd prefer do that in a seperate PR. This one starts to look like a house of card. Each time I modify something, I fear that eveything falls.